src/Microsoft.VisualBasic/Microsoft.VisualBasic.sln

```diff
@@ -7,6 +7,11 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.VisualBasic.Tests
 		{FE25D593-8029-4726-ABC2-944522B5BE04} = {FE25D593-8029-4726-ABC2-944522B5BE04}
 	EndProjectSection
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.VisualBasic.VB.Tests", "tests\VB\Microsoft.VisualBasic.VB.Tests.vbproj", "{da95454e-5f0e-4f8a-871a-25d1ab62867a}"
```

Comment:
I don't think the C# and VB project can have the same Project GUID {9A19103F-16F7-4668-BE54-9A1E7A4F7556} it will not load in VS.

Reply:
You are right. I fixed the tool that was doing that and the guid. It works for me now (albeit with 3 spurious entries in the VS error list)

I also verified that on 15.8 you will not get the popup in VS that you were getting in 15.7.